python/samples/concepts/plugins/crew_ai_plugin.py

```diff
@@ -0,0 +1,75 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+import asyncio
+import logging
+
+from semantic_kernel.core_plugins.crew_ai.crew_ai_enterprise import CrewAIEnterprise, InputMetadata
```

Comment:
make sure that everything a dev would need is importable with:
```suggestion
from semantic_kernel.core_plugins.crew_ai import CrewAIEnterprise, InputMetadata
```